src/bin/sccache-dist/cmdline/parse.rs

```diff
@@ -0,0 +1,361 @@
+// Copyright 2022 <LovecraftianHorror@pm.me>
```

Comment:
I think the copyright is incorrect.
You probably need to keep Mozilla's as you mostly moved content, no?

Reply:
All creds go to @LovecraftianHorror - but from the diff I'd agree.

Reply:
thanks :)